runelite-client/src/main/java/net/runelite/client/plugins/npchighlight/NpcIndicatorsConfig.java

```diff
@@ -118,4 +118,25 @@ default boolean showRespawnTimer()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+		position = 8,
+		keyName = "deadNpcColor",
+		name = "Dead NPC menu entry color",
+		description = "Color of menu entries that refer to dead NPCs"
+	)
+	default Color getdeadNpcColor()
+	{
+		return Color.GRAY;
+	}
+
+	@ConfigItem(
+		position = 9,
+		keyName = "recolorDeadNpcs",
+		name = "Recolor dead NPC menu entries",
+		description = "Changes the color of dead NPCs in menu entries")
+	default boolean recolorDeadNpcs()
```

Comment:
and then delete this..

Reply:
Are you implying that you don't think that the menu entry recoluring should be toggleable off once a colour is selected? I can't see any practical reason to ever turn it off, but it seems a bit annoying to have to reset the entire plugin if someone changes their minds.

Reply:
Uh, is it not possible to select no color with the colorpicker?

Reply:
It's not, but people can just reset the plugin.

Reply:
I did not realize it was not possible to unset it, in which case I don't care if you want this back the other way or not. IMO this is probably better dealt with by making the color picker allow unsetting, though.